src/main/java/appeng/parts/automation/PartImportBus.java

```diff
@@ -284,6 +285,31 @@ private boolean importStuff( InventoryAdaptor myAdaptor, IAEItemStack whatToImpo
 		return false;
 	}
 
+	private int calculateMaximumAmountToImport( InventoryAdaptor myAdaptor, IAEItemStack whatToImport, IMEMonitor<IAEItemStack> inv, FuzzyMode fzMode )
+	{
+		final int toSend = Math.min( this.itemToSend, 64 );
+		final ItemStack simResult;
+		final IAEItemStack itemAmountNotStorable;
+
+		if( this.getInstalledUpgrades( Upgrades.FUZZY ) > 0 )
+		{
+			simResult = myAdaptor.simulateSimilarRemove( toSend, whatToImport == null ? null : whatToImport.getItemStack(), fzMode, this.configDestination( inv ) );
```

Comment:
arguments of both cases can be extracted to the front of the `if`clause, 
would also improve readability